pages/index.js

```diff
@@ -1,4 +1,13 @@
 const html = require('html-template-tag')
+const homepageData = require('../public/data/homepage.json')
+const md = require('markdown-it')()
+
+const Blurb = ({ title, body }) => html`
+  <div class="Blurb">
+    <h2 class="Blurb-title">${title}</h2>
```

Comment:
Also make sure that this is escaped properly.

Reply:
Thanks, will make sure :)